### PR TITLE
Formatters

### PR DESCRIPTION
**Pull request by DawoodMehmood**
- Added comptroller formatters for (grant, approve)
- Added general formatter for (transfer)

### CODE REVIEW

File: checks/compound/formatters/general-formatters.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import { Contract } from 'ethers'
+import { customProvider } from '../../../utils/clients/ethers'
+import { getContractNameAndAbiFromFile } from './../abi-utils'
+import { CometChains, ExecuteTransactionInfo, TransactionFormatter } from './../compound-types'
+import { defactor, getPlatform } from './helper'
+
+export const generalFormatters: { [functionName: string]: TransactionFormatter } = {
+  'transfer(address,uint256)': async (
```
Comment (RobinNagpal) [inline checks/compound/formatters/general-formatters.ts:8]: We can add approve also to this list

File: checks/compound/formatters/helper.ts [hunk 2]
```diff
@@ -46,3 +47,16 @@ export function toYears(seconds: number, secondsPerYear = 31536000): number {
...
+export async function getPlatform(chain: CometChains) {
+  switch (chain) {
+    case CometChains.mainnet:
+      return 'etherscan.io'
+    case CometChains.polygon:
+      return 'polygonscan.com'
+    case CometChains.arbitrum:
+      return 'arbiscan.io'
+    case CometChains.base:
+      return ''
```
Comment (RobinNagpal) [inline checks/compound/formatters/helper.ts:60]: Why do we have this as empty?

File: checks/compound/formatters/general-formatters.ts (hunk 1 above)
Comment (DawoodMehmood) [reply checks/compound/formatters/general-formatters.ts:8]: okay 

got it

File: checks/compound/formatters/helper.ts (hunk 2 above)
Comment (DawoodMehmood) [reply checks/compound/formatters/helper.ts:60]: I havent encountered any proposal relevant to this. just kept it as a placeholder

**Pull request merged**